packages/squiggle-lang/__tests__/TS/JS_test.ts

```diff
@@ -68,6 +68,29 @@ describe("Partials", () => {
   });
 });
 
+describe("Parameters", () => {
+  test("Can pass parameters into partials and cells", () => {
+    let bindings = testRunPartial(`y = $x + 2`, {}, { x: 1 }); // y = 3
+    let bindings2 = testRunPartial(`z = $x + y * $a`, bindings, { a: 3 }); // z = 1 + 3 * 3 = 10
```

Comment:
Reading this test requires me to keep some math in my head. There must be a nicer way of doing this.

Maybe something like,
``
let testVars = {x:1, a: 3}
let calculatedZ = testVars.x  + (testVars.x + 2) * testVars.3
let calculatedResult = calculatedZ + testVars.x + testVars.a + (testVars.x + 2) 
```
...